### ISSUE TITLE
No bookmarks in startify

### ISSUE THREAD

**Issue by danshumaker** [wsdjeg/SpaceVim#2962]
## Expected behavior, english is required
When using the startify home `SPC bh`  the bookmarks I have set in ~/SpaceVim.d/autoload/myspacevim.vim will show up.
 
## The reproduce ways from Vim starting (Required!)
start vim with no options.

## Debug info
(grrr SPC h l doesn't copy to clipboard)  had to cmd-a cmd-c cmd-v
```SpaceVim layers:

+ autocomplete:             loaded          https://spacevim.org/layers/autocomplete/
- chat:                 not loaded          https://spacevim.org/layers/chat/
+ checkers:                 loaded          https://spacevim.org/layers/checkers/
- chinese:              not loaded          https://spacevim.org/layers/chinese/
+ colorscheme:              loaded          https://spacevim.org/layers/colorscheme/
- core#banner:          not loaded          https://spacevim.org/layers/core/banner/
+ core#statusline:          loaded          https://spacevim.org/layers/core/statusline/
+ core#tabline:             loaded          https://spacevim.org/layers/core/tabline/
+ core:                     loaded          https://spacevim.org/layers/core/
- cscope:               not loaded          https://spacevim.org/layers/cscope/
+ ctrlp:                    loaded          https://spacevim.org/layers/ctrlp/
- debug:                not loaded          https://spacevim.org/layers/debug/
- denite:               not loaded          https://spacevim.org/layers/denite/
+ edit:                     loaded          https://spacevim.org/layers/edit/
- exprfold:             not loaded          no exists
- floobits:             not loaded          https://spacevim.org/layers/floobits/
+ format:                   loaded          https://spacevim.org/layers/format/
+ fzf:                      loaded          https://spacevim.org/layers/fzf/
- games:                not loaded          no exists
+ git:                      loaded          https://spacevim.org/layers/git/
+ github:                   loaded          https://spacevim.org/layers/github/
- incsearch:            not loaded          no exists
- indentmove:           not loaded          no exists
- japanese:             not loaded          https://spacevim.org/layers/japanese/
- lang#agda:            not loaded          https://spacevim.org/layers/lang/agda/
- lang#asciidoc:        not loaded          https://spacevim.org/layers/lang/asciidoc/
- lang#autohotkey:      not loaded          https://spacevim.org/layers/lang/autohotkey/
- lang#c:               not loaded          https://spacevim.org/layers/lang/c/
- lang#clojure:         not loaded          https://spacevim.org/layers/lang/clojure/
- lang#coffeescript:    not loaded          https://spacevim.org/layers/lang/coffeescript/
- lang#crystal:         not loaded          no exists
- lang#csharp:          not loaded          https://spacevim.org/layers/lang/csharp/
- lang#d:               not loaded          https://spacevim.org/layers/lang/d/
- lang#dart:            not loaded          https://spacevim.org/layers/lang/dart/
+ lang#dockerfile:          loaded          https://spacevim.org/layers/lang/dockerfile/
- lang#elixir:          not loaded          https://spacevim.org/layers/lang/elixir/
- lang#elm:             not loaded          https://spacevim.org/layers/lang/elm/
- lang#erlang:          not loaded          https://spacevim.org/layers/lang/erlang/
- lang#extra:           not loaded          https://spacevim.org/layers/lang/extra/
- lang#fsharp:          not loaded          https://spacevim.org/layers/lang/fsharp/
- lang#go:              not loaded          https://spacevim.org/layers/lang/go/
- lang#graphql:         not loaded          https://spacevim.org/layers/lang/graphql/
- lang#groovy:          not loaded          https://spacevim.org/layers/lang/groovy/
- lang#hack:            not loaded          https://spacevim.org/layers/lang/hack/
- lang#haskell:         not loaded          https://spacevim.org/layers/lang/haskell/
+ lang#html:                loaded          https://spacevim.org/layers/lang/html/
- lang#hy:              not loaded          https://spacevim.org/layers/lang/hy/
- lang#ipynb:           not loaded          https://spacevim.org/layers/lang/ipynb/
- lang#java:            not loaded          https://spacevim.org/layers/lang/java/
+ lang#javascript:          loaded          https://spacevim.org/layers/lang/javascript/
- lang#json:            not loaded          no exists
- lang#julia:           not loaded          https://spacevim.org/layers/lang/julia/
- lang#kotlin:          not loaded          https://spacevim.org/layers/lang/kotlin/
- lang#latex:           not loaded          https://spacevim.org/layers/lang/latex/
- lang#lisp:            not loaded          https://spacevim.org/layers/lang/lisp/
- lang#livescript:      not loaded          https://spacevim.org/layers/lang/livescript/
- lang#lua:             not loaded          https://spacevim.org/layers/lang/lua/
+ lang#markdown:            loaded          https://spacevim.org/layers/lang/markdown/
- lang#matlab:          not loaded          https://spacevim.org/layers/lang/matlab/
- lang#nim:             not loaded          https://spacevim.org/layers/lang/nim/
- lang#ocaml:           not loaded          https://spacevim.org/layers/lang/ocaml/
- lang#pact:            not loaded          https://spacevim.org/layers/lang/pact/
- lang#pascal:          not loaded          no exists
- lang#perl:            not loaded          https://spacevim.org/layers/lang/perl/
+ lang#php:                 loaded          https://spacevim.org/layers/lang/php/
- lang#plantuml:        not loaded          https://spacevim.org/layers/lang/plantuml/
- lang#pony:            not loaded          https://spacevim.org/layers/lang/pony/
- lang#powershell:      not loaded          https://spacevim.org/layers/lang/powershell/
- lang#processing:      not loaded          https://spacevim.org/layers/lang/processing/
- lang#prolog:          not loaded          https://spacevim.org/layers/lang/prolog/
- lang#puppet:          not loaded          https://spacevim.org/layers/lang/puppet/
- lang#purescript:      not loaded          https://spacevim.org/layers/lang/purescript/
+ lang#python:              loaded          https://spacevim.org/layers/lang/python/
- lang#r:               not loaded          https://spacevim.org/layers/lang/r/
- lang#racket:          not loaded          https://spacevim.org/layers/lang/racket/
- lang#red:             not loaded          https://spacevim.org/layers/lang/red/
- lang#ruby:            not loaded          https://spacevim.org/layers/lang/ruby/
- lang#rust:            not loaded          https://spacevim.org/layers/lang/rust/
- lang#scala:           not loaded          https://spacevim.org/layers/lang/scala/
- lang#scheme:          not loaded          https://spacevim.org/layers/lang/scheme/
- lang#sh:              not loaded          https://spacevim.org/layers/lang/sh/
- lang#slim:            not loaded          no exists
- lang#solidity:        not loaded          no exists
- lang#sql:             not loaded          no exists
- lang#swift:           not loaded          https://spacevim.org/layers/lang/swift/
- lang#swig:            not loaded          no exists
- lang#tcl:             not loaded          https://spacevim.org/layers/lang/tcl/
- lang#toml:            not loaded          no exists
- lang#typescript:      not loaded          https://spacevim.org/layers/lang/typescript/
- lang#v:               not loaded          https://spacevim.org/layers/lang/v/
- lang#vim:             not loaded          https://spacevim.org/layers/lang/vim/
+ lang#vue:                 loaded          https://spacevim.org/layers/lang/vue/
- lang#WebAssembly:     not loaded          https://spacevim.org/layers/lang/WebAssembly/
- lang#xml:             not loaded          no exists
- leaderf:              not loaded          https://spacevim.org/layers/leaderf/
- lsp:                  not loaded          no exists
- mail:                 not loaded          no exists
- operator:             not loaded          no exists
- org:                  not loaded          no exists
+ shell:                    loaded          https://spacevim.org/layers/shell/
- sudo:                 not loaded          https://spacevim.org/layers/sudo/
+ tags:                     loaded          no exists
- test:                 not loaded          https://spacevim.org/layers/test/
- tmux:                 not loaded          https://spacevim.org/layers/tmux/
- tools#dash:           not loaded          https://spacevim.org/layers/tools/dash/
- tools#mpv:            not loaded          no exists
- tools#screensaver:    not loaded          no exists
- tools#zeal:           not loaded          https://spacevim.org/layers/tools/zeal/
+ tools:                    loaded          https://spacevim.org/layers/tools/
+ ui:                       loaded          https://spacevim.org/layers/ui/
- unite:                not loaded          https://spacevim.org/layers/unite/
+ VersionControl:           loaded          https://spacevim.org/layers/VersionControl/
- vim:                  not loaded          no exists
```

## Screenshots
Apologies for this painfully slow recording.  It shows how I've got my bookmarks saved
in the ~/SpaceVim.d/autoload/myspacevim.vim file that do not show up in the SPC bh
screen.
https://asciinema.org/a/itIUCGWUk4YqS2EbNtstYulhK

**Comment by wsdjeg** [comment]
no debug info.

**Comment by danshumaker** [comment]
sorry, including debug info:


## Expected behavior, english is recommend

## Environment Information

- OS: mac
- vim version: 8.1.1517
- neovim version: -
- SpaceVim version: 1.3.0-dev
- SpaceVim status: 

```
On branch master
Your branch is up to date with 'origin/master'.

Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git checkout -- <file>..." to discard changes in working directory)

	modified:   .SpaceVim.d/init.vim

no changes added to commit (use "git add" and/or "git commit -a")
```

## The reproduce ways from Vim starting (Required!)

## Output of the `:SPDebugInfo!`

<details><summary> SpaceVim debug information </summary>

### SpaceVim options :

```toml
  auto_disable_touchpad = 1
  autocomplete_method = 'neocomplete'
  autocomplete_parens = 1
  buffer_index_type = 4
  checkinstall = 1
  colorscheme = 'gruvbox'
  colorscheme_bg = 'dark'
  colorscheme_default = 'desert'
  commandline_prompt = '➭'
  custom_color_palette = []
  custom_plugins = [['rayburgemeestre/phpfolding.vim', {'name': 'rayburgemeestre/phpfolding.vim', 'on_ft': 'php'}]]
  debug_level = 1
  default_indent = 2
  disabled_plugins = []
  enable_ale = 1
  enable_bepo_layout = 0
  enable_cursorcolumn = 0
  enable_cursorline = 1
  enable_debug = 0
  enable_googlesuggest = 0
  enable_guicolors = 1
  enable_insert_leader = 1
  enable_javacomplete2_py = 0
  enable_key_frequency = 0
  enable_language_specific_leader = 1
  enable_neocomplcache = 0
  enable_neomake = 1
  enable_os_fileformat_icon = 0
  enable_powerline_fonts = 1
  enable_statusline_bfpath = 0
  enable_statusline_mode = 1
  enable_statusline_tag = 1
  enable_tabline_filetype_icon = 1
  enable_tabline_ft_icon = 0
  enable_vimfiler_filetypeicon = 0
  enable_vimfiler_gitstatus = 0
  enable_vimfiler_welcome = 1
  enable_ycm = 0
  error_symbol = '✖'
  expand_tab = 1
  filemanager = 'nerdtree'
  filetree_direction = 'left'
  filetype_icons = {}
  force_global_config = 0
  gitcommit_issue_icon = ''
  gitcommit_pr_icon = ''
  github_username = ''
  guifont = 'SauceCodePro Nerd Font Mono:h11'
  hiddenfileinfo = 1
  home_files_number = 6
  hosts_url = 'https://raw.githubusercontent.com/racaljk/hosts/master/hosts'
  info_symbol = 'ⓘ'
  keep_server_alive = 1
  language = ''
  leader_guide_default_group_name = ''
  leader_guide_flatten = 1
  leader_guide_hspace = 5
  leader_guide_max_size = 0
  leader_guide_position = 'botright'
  leader_guide_run_map_on_popup = 1
  leader_guide_sort_horizontal = 0
  leader_guide_submode_mappings = {'<C-C>': 'win_close'}
  leader_guide_vertical = 0
  lint_on_save = 1
  lint_on_the_fly = 0
  max_column = 120
  plugin_bundle_dir = '/Users/dan/.cache/vimfiles/'
  plugin_manager = 'dein'
  plugin_manager_processes = 16
  plugin_name = 'palenight.vim'
  project_rooter_automatically = 1
  project_rooter_patterns = ['.git/', '_darcs/', '.hg/', '.bzr/', '.svn/', 'package.json', '.SpaceVim.d/']
  realtime_leader_guide = 1
  relativenumber = 1
  search_tools = ['rg', 'ag', 'pt', 'ack', 'grep', 'findstr']
  sidebar_direction = ''
  sidebar_width = 30
  simple_mode = 0
  smartcloseignoreft = ['tagbar', 'vimfiler', 'defx', 'SpaceVimRunner', 'SpaceVimREPL', 'SpaceVimQuickFix', 'HelpDescribe', 'VebuggerShell', 'VebuggerTerminal', 'SpaceVimTabsManager']
  smartcloseignorewin = ['__Tagbar__', 'vimfiler:default']
  snippet_engine = 'neosnippet'
  src_root = 'E:\sources\'
  statusline_inactive_separator = 'bar'
  statusline_iseparator = 'nil'
  statusline_left_sections = ['winnr', 'filename', 'major mode', 'syntax checking', 'minor mode lighters', 'vcs', 'hunks']
  statusline_right_sections = ['fileformat', 'cursorpos', 'percentage']
  statusline_separator = 'arrow'
  statusline_unicode_symbols = 1
  terminal_cursor_shape = 2
  trim_trailing_whitespace = 1
  update_retry_cnt = 3
  version = '1.3.0-dev'
  vim_help_language = 'en'
  vimcompatible = 1
  warning_symbol = '⚠'
  wildignore = '*/tmp/*,*.so,*.swp,*.zip,*.class,tags,*.jpg,*.ttf,*.TTF,*.png,*/target/*,.git,.svn,.hg,.DS_Store,*.svg'
  windows_index_type = 3
  windows_leader = ''
  windows_smartclose = 0
```


### SpaceVim layers :

```toml
[[layers]]
  name="autocomplete"
  auto-completion-return-key-behavior='complete'
  auto-completion-tab-key-behavior='cycle'
[[layers]]
  name="checkers"
[[layers]]
  name="format"
[[layers]]
  name="edit"
[[layers]]
  name="ui"
[[layers]]
  name="core"
[[layers]]
  name="core#statusline"
[[layers]]
  name="core#tabline"
[[layers]]
  name="shell"
  default_position='top'
  default_height=30
[[layers]]
  name="fzf"
[[layers]]
  name="lang#php"
[[layers]]
  name="VersionControl"
[[layers]]
  name="lang#javascript"
[[layers]]
  name="tools"
[[layers]]
  name="github"
[[layers]]
  name="ctrlp"
[[layers]]
  name="lang#dockerfile"
[[layers]]
  name="lang#html"
[[layers]]
  name="lang#markdown"
[[layers]]
  name="lang#python"
[[layers]]
  name="lang#vue"
[[layers]]
  name="tags"
[[layers]]
  name="airline"
[[layers]]
  name="gtags"
[[layers]]
  name="git"
  git-plugin='fugitive'
[[layers]]
  name="colorscheme"
[[layers]]
  name="startify"
```


### SpaceVim Health checking :

SpaceVim clipboard support check report:
Checking +clipboard:
      SUCCEED!
SpaceVim environment check report:
Current progpath: Vim(/Applications/MacVim.app/Contents/MacOS/Vim)
version: 801
OS: mac
[shell, shellcmdflag, shellslash]: ['/bin/bash', '-c', 0]
SpaceVim lua support check report:
Checking +lua:
      SUCCEED!
SpaceVim python support check report:
Checking +python3:
      SUCCEED!
Checking +python:
      Failed : to support +python, Please install vim-gik, or build from sources.

### SpaceVim runtime log :

```log
[ SpaceVim ] : logger file  does not exists, only log for current process will be shown!
[ SpaceVim ] [09:20:09] [ Info ] Startup with argv: ['/Users/dan/journal/content/Phase2/2019-07-30_Tuesday.md', '/Users/dan/journal/content/Phase2/2019-07-24_Wednesday.md', '/Users/dan/work/Phase2/PHASE2_PROJECT_NOTES.md', '/Users/dan/NOTES.md']
[ SpaceVim ] [09:20:09] [ Info ] find local conf: /Users/dan/.SpaceVim.d/init.toml
[ SpaceVim ] [09:20:09] [ Info ] loadding cached local conf: /Users/dan/.cache/SpaceVim/conf/_Users_dan__SpaceVim_d_init_toml.json
[ SpaceVim ] [09:20:09] [ Info ] start to apply config [local]
[ SpaceVim ] [09:20:09] [ Info ] SpaceVim server startup at:/tmp/spacevim_vim_server
[ SpaceVim ] [09:20:12] [ Info ] Start to find root for: /Users/dan/journal/content/Phase2/2019-07-30_Tuesday.md
[ SpaceVim ] [09:20:12] [ Info ]         (.git/):/Users/dan/journal/.git/
[ SpaceVim ] [09:20:12] [ Info ]         (package.json):/Users/dan/journal/package.json
[ SpaceVim ] [09:20:12] [ Info ] change to root:/Users/dan/journal
[ SpaceVim ] [09:20:18] [ Info ] change to root:/Users/dan/journal
[ SpaceVim ] [09:20:18] [ Info ] change to root:/Users/dan/journal
[ SpaceVim ] [09:20:57] [ Info ] Start to find root for: /Users/dan/journal/content/phase2/2019-07-30_Tuesday.md
[ SpaceVim ] [09:20:57] [ Info ]         (.git/):/Users/dan/journal/.git/
[ SpaceVim ] [09:20:57] [ Info ]         (package.json):/Users/dan/journal/package.json
[ SpaceVim ] [09:20:57] [ Info ] change to root:/Users/dan/journal
[ SpaceVim ] [09:21:06] [ Info ] Start to find root for: /Users/dan/journal/content/phase2/2019-07-30_Tuesday.md
[ SpaceVim ] [09:21:06] [ Info ]         (.git/):/Users/dan/journal/.git/
[ SpaceVim ] [09:21:06] [ Info ]         (package.json):/Users/dan/journal/package.json
[ SpaceVim ] [09:21:06] [ Info ] change to root:/Users/dan/journal
[ SpaceVim ] [09:26:35] [ Info ] Start to find root for: /Users/dan/journal/content/phase2/2019-07-24_Wednesday.md
[ SpaceVim ] [09:26:35] [ Info ]         (.git/):/Users/dan/journal/.git/
[ SpaceVim ] [09:26:35] [ Info ]         (package.json):/Users/dan/journal/package.json
[ SpaceVim ] [09:26:35] [ Info ] change to root:/Users/dan/journal
[ SpaceVim ] [09:26:39] [ Info ] Start to find root for: 
[ SpaceVim ] [09:26:39] [ Info ]         (.git/):/Users/dan/journal/.git/
[ SpaceVim ] [09:26:39] [ Info ]         (package.json):/Users/dan/journal/package.json
[ SpaceVim ] [09:26:39] [ Info ] change to root:/Users/dan/journal
[ SpaceVim ] [09:26:58] [ Info ] Start to find root for: 
[ SpaceVim ] [09:26:58] [ Info ]         (.git/):/Users/dan/journal/.git/
[ SpaceVim ] [09:26:58] [ Info ]         (package.json):/Users/dan/journal/package.json
[ SpaceVim ] [09:26:58] [ Info ] change to root:/Users/dan/journal
[ SpaceVim ] [09:40:07] [ Info ] Start to find root for: /var/folders/fs/qrqy74xd5y388jzfxbtl11vw0000gn/T/v83scpe/25/issue_report.md
[ SpaceVim ] [09:40:07] [ Info ] change to root:/Users/dan/journal
```
</details>

## Screenshots

If you have any screenshots for this issue please upload here. BTW you can use https://asciinema.org/ for recording video  in terminal.

**Comment by danshumaker** [comment]
FWIW, I got the book total file list value working by adding the home_files_number value (20) to my init.toml file.  However, no such luck with bookmarks, I tried adding the home_bookmarks, home_files_bookmarks to the options section of the init.toml file and to the [[layers]] startify section and it didn't help.  As a workaround I'm using the Nerdtree booksmarks but would like to use startify's bookmark list if possible.

In my old vim setup I would just add a key map to my .vimrc to act as the desired "bookmark" (hotkey) but haven't figured out the proper way to do that in the Spacevim.d/autoload/myspacevim.vim file yet.
